apps/demos/Demos/Scheduler/CellTemplates/Angular/app/app.component.ts

```diff
@@ -44,12 +44,35 @@ export class AppComponent {
 
   currentView = this.views[0];
 
+  ariaDescription = () => {
+    const disabledDates = this.holidays.map(date => {
+        if (this.isWeekend(date)) {
+          return null;
+        }
+        return new Date(date).toLocaleDateString('en-US', {
+          weekday: 'long',
+          year: 'numeric',
+          month: 'long',
+          day: 'numeric',
+        })
+      }
+    ).filter(dateText => dateText);
```

Comment:
I suggest to filter records before the using. It will make the code better:
.filter(!this.isWeekend(date))
.map(date => {
   return new Date(date).toLocaleDateString('en-US', {
   ...
})